src/yoroi-wallets/utils/parsing.ts

```diff
@@ -14,26 +13,26 @@ export class InvalidAssetAmount extends ExtendableError {
     TOO_MANY_DECIMAL_PLACES: 'TOO_MANY_DECIMAL_PLACES',
     TOO_LARGE: 'TOO_LARGE',
     TOO_LOW: 'TOO_LOW',
-    LT_MIN_UTXO: 'LT_MIN_UTXO', // amount is less than min utxo value allowed
+    LT_MIN_UTXO: 'LT_MIN_UTXO',
+    // amount is less than min utxo value allowed
     NEGATIVE: 'NEGATIVE',
   }
 
-  constructor(errorCode: $Values<typeof InvalidAssetAmount.ERROR_CODES>) {
+  constructor(errorCode: typeof InvalidAssetAmount.ERROR_CODES[keyof typeof InvalidAssetAmount.ERROR_CODES]) {
     super('InvalidAssetAmount')
-    this.errorCode = errorCode
+    ;(this as any).errorCode = errorCode
```

Comment:
```suggestion
    (this as any).errorCode = errorCode
```

Reply:
the semicolon is added automatically, it cant be removed